stage0_bin/supply-chain/audits.toml

```diff
@@ -82,6 +82,12 @@ criteria = "safe-to-deploy"
 version = "0.4.0"
 notes = "This crate does not have any unsafe code and does not use any ambient capabilities such as filesystem or network access."
 
+[[audits.lock_api]]
+who = "Juliette Pretot <julsh@google.com>"
+criteria = ["safe-to-deploy", "does-not-implement-crypto"]
+version = "0.4.9"
+notes = "Limited unsafe code that is well commented to explain safety. No ambient capabilities. No crypto implementation."
```

Comment:
In addition to the comments is the code itself reasonable? :)

Reply:
good catch. it is, forgot to comment that in notes.